hera_cal/abscal.py

```diff
@@ -1013,12 +1013,14 @@ def cross_pol_phase_cal(model, data, model_bls, data_bls, wgts={}, return_gains=
         Default is False.
     gain_ants : list of tuples
         List of antenna-pol tuples to return gains for. Default is None.
+    refpol : str
+        Reference polarization to use for relative phase calibration. Default is 'Jee'.
```

Comment:
I'd be even more explicit: gains in the `refpol` are not change by relative polarization calibration.

Reply:
Also, do we want to do any checking that one of the two pols is the refpol and that there are only two pols?

Reply:
Added these checks